### PR TITLE
Use 75% potion

### PR DESCRIPTION
**Pull request by aosyatnik**
I couldn't find anything, that makes these potions different from any other. SO I just hardcoded the right numbers of type and type id.

### CODE REVIEW

**Comment by matigramirez** [comment]
I checked the items that restore HP, SP, MP based on a % and they all are `Special = 17`. 

image

Potions with constant values seem to be `Special = 0`. Keep in mind that normal non-consumable items also are `Special = 0` with the difference that potions have `ReqIg != 0` (`ReqIg` stands for `Spell group` in this case). Basically, constant potions don't have a special effect but they do have a spell group, this is the key difference.

`ReqIg = 3` for HP potions and `ReqIg = 2` for MP, SP potions. I'm not sure this is the case for all the potions but I spent a fair amount of time going trough Item.SData and didn't find any differences with what I'm saying.

To sum up:
Percentage Potions -> `Special = 17`
Constant Potions -> `Special = 0` & `ReqIg = 2 or 3)`

**Comment by matigramirez** [comment]
On the source code, `Special = 17` triggers the `UseHealingPotion` method which is intended to be used by both constant and percentage potions. We should change this and have 2 different methods.

**Comment by aosyatnik** [comment]
@matigramirez , I have some doubts about `Special = 17`, that it's only for %-potions. If you check for example type 25, type id 19, which is "Mini Healing Potion", it also has Special 17, but it doesn't heal %, it heals 1057 hp. So, Special == 17 doesn't mean, that it's definitely %-potion...

Regarding `ReqIg` you are partially right, I also noticed, that 3 is for hp, 2 for mp/sp. 4 is for hp&mp&sp. Problem is that "Super potion" (25, 38), that heals 1500 hp, mp, sp has ReqIg == 4, the same as "Etain Potion" (100, 1). So I'm kind of confused. I also thought, that maybe ReqIg is item cool down. What do you think?

**Review by matigramirez — APPROVED** [review]
Everything looks good, I tested potions in-game and they work as intended.
Maybe we should add some tests for constant potions aswell?

**Comment by aosyatnik** [comment]
Right, added test for red apple.
